IDsAboveHead/cl_idsabovehead.lua

```diff
@@ -1,4 +1,14 @@
-local disPlayerNames = 5
+displayid = true
+RegisterCommand(Config.ToggleCommand, function(source, args, rawCommand)
+    if displayid == true then

```

Comment:
This whole command can be cleaned up

```lua
displayid = not displayid
TriggerEvent("chatMessage", "", {255,255,255}, Config.Prefix.." ^3You have ^5"..displayid and "enabled" or "disabled".." ^3IDs above head.")
```

Reply:
Thank you for that. That is very helpful and will take note of that. I appreciate it.

Reply:
SCRIPT ERROR: @fivem-toggleid/client.lua:5: attempt to concatenate a boolean value (global 'displayid')
ref (@fivem-toggleid/client.lua:5)

I tested that piece of code and it gave me an error btw.

Reply:
```lua
displayid = not displayid
TriggerEvent("chatMessage", "", {255,255,255}, Config.Prefix.." ^3You have ^5"..(displayid and "enabled" or "disabled").." ^3IDs above head.")
```
*should* fix that
forgot you need brackets when doing shit like that

Reply:
That worked! That's actually sick. Good to know.

Reply:
`chatMessage` is deprecated and you should use `chat:addMessage` instead, [Docs reference](https://docs.fivem.net/docs/resources/chat/events/chatMessage/)